components/organisms/home/HomeDiscoverPartners.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="relative pb-48 sm:pb-56 md:pb-64 lg:pb-72 xl:pb-92 2xl:pb-128 bg-sky-surface">
-    <section class="relative pt-20">
+    <section class="relative pt-20 z-20">
```

Comment:
You could reduce it to `z-10` and remove `z-10` from the below `img`. It would simplify things and make sure z-indexes don't escalde too quickly.

Reply:
This is definitely a better solution! :)